src/app/static/src/tests/components/runReport/runReport.test.ts

```diff
@@ -568,4 +568,72 @@ describe("runReport", () => {
         expect(wrapper.vm.$data.runningStatus).toBe("");
         expect(wrapper.find("#run-form-group button").attributes("disabled")).toBeUndefined();
     });
+
+    it("it does render changelog message and type correctly if a report is selected", () => {
+        const changelogTypes =  ["internal", "public"]
+        const wrapper = mount(RunReport, {
+            propsData: {
+                metadata: {
+                    git_supported: true,
+                    changelog_types: changelogTypes
+                },
+                gitBranches
+            },
+            data() {
+                return {
+                    selectedReport: "report"
+                }
+            }
+        });
+
+        expect(wrapper.find("#changelog-message").exists()).toBe(true);
+        const options = wrapper.find("#changelog-type").find("select")
+            .findAll("option")
+        expect(options.at(0).text()).toBe(changelogTypes[0]);
+        expect(options.at(1).text()).toBe(changelogTypes[1]);
+    });
+
+    it("it does not render changelog message and type if a report is not selected", () => {
+        const changelogTypes =  ["internal", "public"]
+        const wrapper = mount(RunReport, {
+            propsData: {
+                metadata: {
+                    git_supported: false,
+                    changelog_types: changelogTypes,
+                    selectedReport: "report"
```

Comment:
This line is confusing and should be removed - `metadata` will never include `selectedReport`, and this test is about the case where a report _isn't_ seletced.